packages/migrate/src/utils/getSchemaPathAndPrint.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+import chalk from 'chalk'
+import { getSchemaPath, logger } from '@prisma/sdk'
+import { NoSchemaFoundError } from './errors'
+
+// TODO move NoSchemaFoundError to sdk and this too
+// then replace the 2 hardcoded errors to NoSchemaFoundError in
+// https://github.com/prisma/prisma/blob/main/packages/sdk/src/cli/getSchema.ts#L383:L383
```

Comment:
Better to link to the file in a stable commit, otherwise the line numbers will become wrong when the file is changed in `main`.

```suggestion
// https://github.com/prisma/prisma/blob/bbdf1c23653a77b0b5bf7d62efd243dcebea018b/packages/sdk/src/cli/getSchema.ts#L383:L383
```